### ISSUE TITLE
Keyboard navigation cannot move through disabled items

### ISSUE THREAD

**Issue by tapopov** [telerik/kendo-ui-core#901]
Steps to reproduce:
1. Focus the Menu
2. Keep navigating to the right

Expected results:
The navigation should proceed to the next available menu item.

Actual results:
Navigation stops at the first disabled item. Navigating to the left does not seem to be affected.

[Example](http://dojo.telerik.com/@popov/obIVi).

**Comment by nstoychev** [comment]
Not reproducible using 2017.2.621 version. 
Dojo: [http://dojo.telerik.com/@nstoychev/OciSO](http://dojo.telerik.com/@nstoychev/OciSO)

**Comment by Xizario** [comment]
@nstoychev 

Using latest version:
http://dojo.telerik.com/oSaBo
When you click on the last item, or select the menu by pressing Tab until it is in focus.

Pressing Left key, rolls the entire menu, including the disabled items
Pressing Right key, rolls until the next disabled item, and then do nothing,

**Comment by nstoychev** [comment]
@Xizario

Thank you for the feedback. I missed navigating by arrow keys.
